rskj-core/src/main/java/co/rsk/cli/tools/DbMigrate.java

```diff
@@ -103,32 +110,52 @@ protected void onExecute(@Nonnull String[] args, @Nonnull RskContext ctx) throws
             throw new IllegalArgumentException("Db to migrate not specified. Please specify in the first argument.");
         }
 
-        DbKind sourceDbKind = ctx.getRskSystemProperties().databaseKind();
+        DbKind defaultDbKind = ctx.getRskSystemProperties().databaseKind();
         DbKind targetDbKind = DbKind.ofName(args[0]);
 
-        if (sourceDbKind == targetDbKind) {
-            throw new IllegalArgumentException(String.format(
-                    "Cannot migrate to the same database, current db is %s and target db is %s",
-                    sourceDbKind,
-                    targetDbKind
-            ));
-        }
-
         String sourceDbDir = ctx.getRskSystemProperties().databaseDir();
         String targetDbDir = sourceDbDir + "_tmp";
```

Comment:
`targetDbDir` is a subfolder of `sourceDbDir`. Won't `FileUtil.recursiveDelete(sourceDbDir)` delete both `targetDbDir` and `sourceDbDir`? Could you test this?

Reply:
I already rebased from master

Reply:
got it 👍